lib/NotificationAdapterMandrill.php

```diff
@@ -48,7 +48,13 @@ public function getErrorMessage() {
     public function __construct() {
         try {
 
-            $this->mandrill = new \Mandrill( \get_option( 'rplus_notifications_adapters_mandrill_apikey' ) );
+			// When api key is defined via constant, take that.
```

Comment:
![Take That](https://media.giphy.com/media/9sliq6q3jqLbq/giphy.gif)

Reply:
?